usr/lib/config/configuration.h

```diff
@@ -65,18 +65,26 @@
  */
 #define CT_BARECONST    (1u << 10u)
 /*
- * A bare number constant, i.e., a bare number outside of a list that
+ * A pair of numbers
+ */
+#define CT_NUMPAIR      (1u << 11u)
+/*
+ * A list of number pairs
+ */
+#define CT_NUMPAIRLIST  (1u << 12u)
+/*
+ * A bare quoted string constant, i.e., a bare string outside of a list that
  * represents its own configuration element.
  */
-#define CT_BARENUMCONST (1u << 11u)
+#define CT_BARESTRINGCONST (1u << 13u)
 
 /*
  * Mask for all types that have a key.  This excludes FILEVERSION,
  * EOC, and BARE which reuse the key field for the value.
  */
 #define CT_HAS_KEY_MASK (CT_INTVAL | CT_STRINGVAL | CT_VERSIONVAL |  \
 			 CT_BAREVAL | CT_STRINGVAL | CT_IDX_STRUCT | \
-			 CT_BARELIST | CT_BARECONST)
+			 CT_BARELIST | CT_BARECONST | CT_NUMPAIRLIST | CT_BARESTRINGCONST)
```

Comment:
This looks wrong.  Does a bare string really have an own key?  I thought the key field is reused to store the bare string.  On the other hand, bare consts are in the list.  Should they be in there?

Reply:
Well, having it in there makes it possible to `confignode_find` the node by its key string.  So maybe that was intended.

Reply:
>having it in there makes it possible to confignode_find the node by its key string 

Yes I guess that was the intention.